src/main.cc

```diff
@@ -16,6 +16,31 @@ namespace {
 
 const DWORD MAX_VALUE_NAME = 16383;
 
+LPWSTR utf8ToWideChar(std::string utf8) {
```

Comment:
For reference, this snippet was crimped from [`node-keytar`](https://github.com/atom/node-keytar): https://github.com/atom/node-keytar/blob/master/src/keytar_win.cc#L10-L33